webapp/package.json

```diff
@@ -24,19 +23,29 @@
     "not op_mini all"
   ],
   "dependencies": {
+    "@apollo/react-hooks": "^4.0.0",
     "@eoscostarica/eoscr-components": "~1.1.5",
```

Comment:
should we update this?

Reply:
yes, you are right I will update it in my next PR